druntime/src/core/internal/gc/impl/conservative/gc.d

```diff
@@ -3600,21 +3569,25 @@ Lmark:
         }
         assert(pbot < ptop);
 
-        busyThreads.atomicOp!"+="(1); // main thread is busy
-
-        evStart.setIfInitialized();
+        evStackFilled.setIfInitialized(); // background threads start now
 
         debug(PARALLEL_PRINTF) printf("mark %lld roots\n", cast(ulong)(ptop - pbot));
 
+        void pullLoop(bool precise)()
+        {
+            static if (precise)
+                mark!(true, true, true)(ScanRange!true(pbot, ptop, null));
+            else
+                mark!(false, true, true)(ScanRange!false(pbot, ptop));
```

Comment:
```suggestion
            mark!(precise, true, true)(ScanRange!precise(pbot, ptop));
```
`null` is the default for precise bitmap, so we don't need this added complication.